generated/nidaqmx/_library_interpreter.py

```diff
@@ -6446,7 +6450,8 @@ def set_wfm_attr_callback(
             value: ExtendedPropertyValue,
```

Comment:
Don't allocate t0_array and dt_array when timing is disabled.

That's what the `0 if t0_array is None else t0_array.size` is for.